sysdump/sysdump.go

```diff
@@ -26,6 +25,8 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 
+	"github.com/cilium/cilium/pkg/versioncheck"
```

Comment:
I think it should stay in the previous section because it doesn't have a prefix "github.com/cilium/cilium" (in fact, workerpool lives in its own [repository](https://github.com/cilium/workerpool)).
At least this is the default settings we use for organizing imports in cilium (my IDE is automatically moving it). Are we are using something different in cilium-cli?

Reply:
Nit: in cilium/cilium and cilium/cilium-cli we AFAIK only use three import groups:

1. standard libary
2. other 3rd party imports
3. imports from the same module

Following the above, I think the `github.com/cilium/cilium/pkg/versioncheck` import should stay where it currently is because that package is a 3rd party import from the perspective of this module (`github.com/cilium/cilium-cli/sysdump`)

Reply:
Might be some misconfig in my IDE, then. I'll move it back there :+1: